python/lib/sift_py/data_import/_config_test.py

```diff
@@ -0,0 +1,232 @@
+import pytest
+from sift_py.data_import.config import CsvConfig
+
+
+def test_empty_data_columns():
+    with pytest.raises(Exception, match="Empty 'data_columns'"):
+        CsvConfig(
+            {
+                "asset_name": "test_asset",
```

Comment:
Super small nit: You could consider making a fixture for this just to save a bit of copy-pasta, and then editing the `data_columns` or other properties as needed to trigger different raises and such. Could just make it a bit easier to update down the line if we change the shape of or add fields to `CsvConfig`

Reply:
Good call. Will update this since it makes it easier to see what changed.